src/image/pixels.js

```diff
@@ -441,6 +441,9 @@ p5.prototype.filter = function(operation, value) {
  * <br><br>
  * See the reference for <a href="#/p5/pixels">pixels[]</a> for more information.
  *
+ * If you want to extract an array of colors or a subimage from an p5.Image object,
+ * take a look at <a href="#/p5.Image/get">p5.Image.get</a>
```

Comment:
this is great, can you update the name to p5.Image.get()
(with parens)

thanks!